integrations/github/src/webhooks.ts

```diff
@@ -147,6 +151,10 @@ export async function handlePullRequestEvents(
                         standalone: {
                             ref: headRef,
                         },
+                        eventCreatedAt:
```

Comment:
I'm wondering if we should add it here.
If I'm not wrong, those are for PR previews, so they are standalone and don't need to be ordered.
CC @taranvohra Wdyt?

Reply:
Yeah let's avoid it for PR previews

Reply:
Removed it for previews.